innodb-java-reader/src/main/java/com/alibaba/innodb/java/reader/util/ZlibUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.alibaba.innodb.java.reader.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.zip.Inflater;
+
+/**
+ * @author jie xu
+ * @description
+ * @created 2020/12/22
+ **/
+public class ZlibUtil {
+  public static byte[] decompress(byte[] data) {
+    byte[] output = new byte[0];
```

Comment:
why create a new output? seems not in use next.

Reply:
you are right, decompress function is copy from network....i will refactor